src/wrapper.cpp

```diff
@@ -196,11 +211,17 @@ bool Wrapper::setDataPointers() {
     Time_1->SetAddress(&ti.time_c);
 	
 	
-    TBranch
-      *ACC_x= tree->GetBranch("gantry0_x"), *g0Y = tree->GetBranch("gantry0_y"), *g0Z = tree->GetBranch("gantry0_z"),
-        *ACC_y = tree->GetBranch("gantry0_rot"), *g0Phi = tree->GetBranch("gantry0_tilt"),
-      *ACC_z = tree->GetBranch("gantry1_x"), *g1Y = tree->GetBranch("gantry1_y"), *g1Z = tree->GetBranch("gantry1_z"),
-        *g1Theta = tree->GetBranch("gantry1_rot"), *g1Phi = tree->GetBranch("gantry1_tilt"),
+	 TBranch
```

Comment:
This doesn't compile for me.  I don't think type of initialization is legal.  Better for me would be 

-        TBranch
-     *ACC_x= tree->GetBranch("phidgACC_Ax");
+     TBranch *ACC_x= tree->GetBranch("phidgACC_Ax");
         ACC_x->SetAddress(&ACC.acc_x);
-     *ACC_y= tree->GetBranch("phidgACC_Ay");
+     TBranch *ACC_y= tree->GetBranch("phidgACC_Ay");
         ACC_y->SetAddress(&ACC.acc_y);
-     *ACC_z= tree->GetBranch("phidgACC_Az");
+     TBranch *ACC_z= tree->GetBranch("phidgACC_Az");
         ACC_z->SetAddress(&ACC.acc_z);